packages/react/src/components/Table/StatefulTable.test.jsx

```diff
@@ -128,10 +117,12 @@ describe('stateful table with real reducer', () => {
     );
     expect(screen.getByText('can pinocchio whiteboard 4B-2-B')).toBeTruthy();
     userEvent.click(screen.getByTestId(`${tableId}-row-4_B-2-B-row-actions-cell-overflow`));
-    userEvent.click(screen.getByText('Add'));
+    await act(async () => {
+      const addBtn = await screen.findByText('Add');
+      userEvent.click(addBtn);
+    });
```

Comment:
This needs to be wrapped in act because of the async nature of the dispatch used in the stateful table, or we get an error about state changes outside of act in the jest tests.